Package.swift

```diff
@@ -1,18 +1,23 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
```

Comment:
This forces consumers to use the current toolchain. 
Is there a requirement for that? (= are we using features of the 5.3 SPM or package format?)

Reply:
No, will switch back to 5.1 here